nidirect_contacts/nidirect_contacts.libraries.yml

```diff
@@ -0,0 +1,6 @@
+gmaps_lazy_load:
+  js:
+    js/gmaps-lazy-load.js: {}
+  dependencies:
+    - core/jquery
+    - core/drupalSettings
```

Comment:
should this depend on geolocation as well ?

Reply:
The formatter includes the required geolocation libraries